src/app/components/Curation/HierarchicalGrid/index.tsx

```diff
@@ -74,7 +74,7 @@ const HiearchicalGrid = ({
             (promo.type === 'video' && `${videoTranslation}, `) ||
             (promo.type === 'photogallery' && `${photoGalleryTranslation}, `);
 
-          const isLive = promo.link?.includes('/live/');
+          const isLive = promo.isLive;
```

Comment:
Just a lint warning, but otherwise all good
```suggestion
          const { isLive } = promo;
```

Reply:
Fixed 👍